courses/machine_learning/deepdive2/feature_engineering/solutions/2_bqml_adv_feat_eng.ipynb

```diff
@@ -0,0 +1,2308 @@
+{
```

Comment:
<p>Delete # Solution</p><p> </p><p>What's the story behind the commented code? If the code isn't part of the optimal solution, it should be deleted.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Deleted the # Solution comment from Lab01 and Lab02 and will remove it from Lab03 and Lab04.